packages/trpc/server/routers/viewer/organizations/getMembers.handler.ts

```diff
@@ -0,0 +1,42 @@
+import { prisma } from "@calcom/prisma";
+
+import type { TrpcSessionUser } from "../../../trpc";
+import type { TGetMembersInputSchema } from "./getMembers.schema";
+
+type CreateOptions = {
+  ctx: {
+    user: NonNullable<TrpcSessionUser>;
+  };
+  input: TGetMembersInputSchema;
+};
+
+export const getMembersHandler = async ({ input, ctx }: CreateOptions) => {
+  const { teamIdToExclude } = input;
+
+  if (!ctx.user.organizationId) return null;
+
+  const users = await prisma.membership.findMany({
+    where: {
+      user: {
+        organizationId: ctx.user.organizationId,
+      },
+      ...(teamIdToExclude && {
+        teamId: {
+          not: teamIdToExclude,
+        },
+      }),
+    },
+    include: {
+      user: {
+        select: {
+          id: true,
+          username: true,
+          email: true,
+          completedOnboarding: true,
+        },
+      },
+    },
+  });
```

Comment:
Get all members from the ORG

Optional get all users that are not in team ID (Will be used in org import for other teams)